### ISSUE TITLE
Security news images lack descriptive alt text

### ISSUE THREAD

**Issue by vbrown608** [EFForg/sec#367]
The alt text is coming from the filename. We could include the correct alt text it in the feed from kittens.

:candy: :candy: :candy:
